### ISSUE TITLE
Revert firmware patch

### ISSUE THREAD

**Issue by fernandobonfimandrade** [metroid-maniac/SCFW#23]
I patched the supercard's firmware with SCFW.frm, but the games I already had on the card no longer run. Do I need to put the roms without the supercard software patch? Is there a possibility to go back to the original firmware?

**Comment by OmDRetro** [comment]
TL;DR: Yes, by default you shouldn't need the patcher, but if you want to use the pre-patched binaries you'll need to disable the `waitstate patching` via kernel settings and rename the ROM extension from `*BIN` to `*GBA`.

There are [special cases](https://github.com/metroid-maniac/SCFW/issues/10) where you do need to use the patcher though, but the instructions provided in that link assumes that your kernel settings are set to its default values(waitstate 1).

***

Reverting back to stock is as simple as [flashing the stock firmware within SCFW](https://github.com/metroid-maniac/SCFW/issues/18)

**Comment by OmDRetro** [comment]
@fernandobonfimandrade , If you have any other questions, please write them down. Otherwise, I'll close this after 4 days.
